products/caasp/main.pm

```diff
@@ -124,6 +124,7 @@ sub load_feature_tests {
     # Container Tests
     if (!check_var('SYSTEM_ROLE', 'microos')) {
         loadtest 'console/docker';
+        loadtest 'console/skopeo' if is_caasp('4.0+');
```

Comment:
Will be skopeo part of CaaSP 4.0 or installed additionally? We will test unregistered CaaSP 4.0 first.

Reply:
I guess this is the same issue as above.

```
loadtest 'console/skopeo' if !is_caasp('3.0+');
```

would that work?

Reply:
You can use is_caasp('qam') if you want to schedule only for registered clusters.